.travis.yml

```diff
@@ -7,6 +7,7 @@ branches:
     - master
 cache:
   directories:
+    - $HOME/$TRAVIS_BUILD_ID
```

Comment:
When does this cache get cleaned up? ¯\_(ツ)_/¯ I assume travis-ci has a way to deal with old caches.
It will only ever be accessed by the current build.

Reply:
https://docs.travis-ci.com/user/caching/#caches-expiration